app/javascript/controllers/search_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
why there are two new controllers?  that do relative the same thing (show/hide blocks)